bindings/node/lib/mongocryptdManager.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+const spawn = require('child_process').spawn;
+const readFile = require('fs').readFile;
+
+/**
+ * @typedef AutoEncryptionExtraOptions
+ * @prop {string} [mongocryptdURI] overrides the uri used to connect to mongocryptd
+ * @prop {boolean} [mongocryptdBypassSpawn=false] if true, autoEncryption will not spawn a mongocryptd
+ * @prop {string} [mongocryptdSpawnPath] the path to the mongocryptd executable
+ * @prop {string[]} [mongocryptdURI] command line arguments to pass to the mongocryptd executable
+ */
+
+module.exports = function() {
+  const platform = require('os').platform;
+
+  const MONGOCRYPTD_PID_FILE = 'mongocryptd.pid';
+  const STATE = {
+    NO_PID_FILE: Symbol('NO_PID_FILE'),
```

Comment:
bikeshed: I've been considering using the node core convention for constants and using `kPrefixedValues` (this actually just comes from Google's C++ style guide, and subsequently V8 I think).

Reply:
that sounds too much like hungarian notation to me :-/. I'd prefer to either keep them as all uppercase, or follow the [airbnb convention](https://github.com/airbnb/javascript#naming--uppercase), which in this case says it should just be camelCase.